clients/ts/signalr/src/HttpConnection.ts

```diff
@@ -17,14 +17,14 @@ const enum ConnectionState {
     Disconnected,
 }
 
-interface INegotiateResponse {
-    connectionId: string;
-    availableTransports: IAvailableTransport[];
-    url: string;
-    accessToken: string;
+export interface INegotiateResponse {
```

Comment:
These are used by some tests now. This file is not exported from `index.ts` though so this has no effect on the public API. Plus, the only changes are to interfaces which don't actually show up in the generated JavaScript.